source/NVDAObjects/__init__.py

```diff
@@ -421,9 +421,8 @@ def _get_name(self):
 	#: Type definition for auto prop '_get_role'
 	role: int

```

Comment:
This is not correct anymore as the type of the role has changed to `controlTypes.Role`